
##### Which statements are true?
5–1 The majority of cells in your body have exactly the
same nucleotide sequence in their genomes.
- True

5–2 In a replication bubble, the same parent DNA
strand serves as the template strand for leading-strand
synthesis at one replication fork and as the template for
lagging-strand synthesis at the other fork.
- True

5–3 In E. coli, where the replication fork travels at
500 nucleotide pairs per second, the DNA ahead of the
fork—in the absence of topoisomerase—would have to
rotate at nearly 3000 revolutions per minute.
- 500 / 10.4 * 60 = 2.884,61538462
- True because DNA turns every 10.4 base-pairs

5–4 When bidirectional replication forks from adja-
cent origins meet, a leading strand always runs into a
lagging strand.
- True, because what is the leading strand from one direction is the lagging strand from the other

5–5 DNA repair mechanisms all depend on the cell
having two homologous chromosomes.
- False. Many mechanisms don't require the homologous chromosome. Only homologous recombination sometimes uses the homologous chromosome, but even it preferably uses the freshly copied double strand of the same chromosome.
##### Discuss the following problems.
5–6 To determine the reproducibility of mutation fre-
quency measurements, you do the following experiment.
You inoculate each of 10 cultures with a single E. coli
bacterium, allow the cultures to grow until each contains
106 cells, and then measure the number of cells in each
culture that carry a mutation in your gene of interest. You
were so surprised by the initial results that you repeated
the experiment to confirm them. Both sets of results dis-
play the same extreme variability, as shown in Table Q5–1.
Assuming that the rate of mutation is constant, why do you
suppose there is so much variation in the frequencies of
mutant cells in different cultures?
- The number of mutated cells is so different because depending on the time the mutation occurred, the mutated cell also divided and produced more of itself. Because this process is exponential, it makes a big difference at which time the mutation happened

5–7 Discuss the following statement: “Primase is a
sloppy enzyme that makes many mistakes. Eventually, the
RNA primers it makes are replaced with DNA made by a
polymerase with higher fidelity. This is wasteful. It would
be more energy-efficient if a DNA polymerase made an
accurate copy in the first place.”
- The reason why DNA polymerase doesn't make its own primer is related to its fidelity. To make it possible that DNA polymerase puts the right nucleotide in the vast amount of cases, complicated mechanisms like proofreading must be in place which don't work if the polymerase makes its own primer. A polymerase that makes its own primer can't be as precise.

5–8 If DNA polymerase requires a perfectly paired
primer in order to add the next nucleotide, how is it that
any mismatched nucleotides “escape” this requirement
and become substrates for mismatch repair enzymes?
- This is extremely rare, DNA polymerase is not perfect. It is possible for a mismatched nucleotide to pair and evade detection by chance.

5–9 DNA repair enzymes preferentially repair mis-
matched bases on the newly synthesized DNA strand,
using the old DNA strand as a template. If mismatches
were instead repaired without regard for which strand
served as template, would mismatch repair reduce repli-
cation errors? Would such a mismatch repair system result
in fewer mutations, more mutations, or the same number
of mutations as there would have been without any repair
at all? Explain your answers.
- It would still decrease mutations, but only in 50% of cases. In the other 50% it would fix the mutation in place by fixing the side that is actually correct. This means fewer mutations overall, but significantly more mutations than strand detecting mismatch repair.

5–10 The laboratory you joined is studying the life cycle
of an animal virus that uses circular, double-stranded
DNA as its genome. Your project is to define the location
of the origin(s) of replication and to determine whether
replication proceeds in one or both directions away from
an origin (unidirectional or bidirectional replication). To
accomplish your goal, you broke open cells infected with
the virus, isolated replicating viral genomes, cleaved them
with a restriction nuclease that cuts the genome at only
one site to produce a linear molecule from the circle, and
examined the resulting molecules in the electron micro-
scope. Some of the molecules you observed are illustrated
schematically in Figure Q5–1. (Note that it is impossible to
distinguish the orientation of one DNA molecule relative
to another in the electron microscope because they land
on the electron microscope grid in random orientations.)
You must present your conclusions to the rest of
the lab tomorrow. How will you answer the two questions
your advisor posed for you? Is there a single, unique origin
of replication or several origins? Is replication unidirec-
tional or bidirectional?
- It is bidirectional, and there is a single unique origin of replication.

5–11 You are investigating DNA synthesis in tissue-
culture cells, using 3H-thymidine to radioactively label
the replication forks. By breaking open the cells in a way
that allows some of the DNA strands to be stretched out,
very long DNA strands can be isolated intact and exam-
ined. You overlay the DNA with a photographic emulsion,
and expose it for 3–6 months, a procedure known as
autoradiography. Because the emulsion is sensitive to
radioactive emissions, the 3H-labeled DNA shows up as
tracks of silver grains. Because the stretching collapses
replication bubbles, the daughter duplexes lie side by side
and cannot be distinguished from each other
You treat the cells to arrest them at the beginning
of S phase. In the first experiment, you release the arrest
and add 3H-thymidine immediately. After 30 minutes, you
wash the cells and change the medium so that the total
concentration of thymidine is the same as it was, but only
one-third of it is radioactive. After an additional 15 min-
utes, you prepare DNA for autoradiography. The results of
this experiment are shown in Figure Q5–2A. In the second
experiment, you release the arrest and wait 30 minutes
before adding 3H-thymidine. After 30 minutes in the pres-
ence of 3H-thymidine, you once again change the medium
to reduce the concentration of radioactive thymidine and
incubate the cells for an additional 15 minutes. The results
of the second experiment are shown in Figure Q5–2B.
A. Explain why, in both experiments, some regions
of the tracks are dense with silver grains (dark), whereas
others are less dense (light).
- The different intensities result from how much radioactive 3H-thymidine was in the solution during replication. if there was a lot, the newly created strands will give up wore radioactive signal and therefore be darker. If there is less, it's the opposite
B. In the first experiment, each track has a central
dark section with light sections at each end. In the second
experiment, the dark section of each track has a light sec-
tion at only one end. Explain the reason for this difference.
- The first experiment looks like this because replication started in the strongly labeled environment, progressing in both directions. When the solution got diluted, both sides continued to replicate with less radioactive 3H-thymidine now, creating lighter edges
- The second experiment shows half-light and half dark parts because replication already started before any radioactive solution was introduced. This leads to an invisible replication origin, followed by strongly labeled parts on both sides, followed by weakly labeled parts on their ends
C. Estimate the rate of fork movement (μm/min) in
these experiments. Do the estimates from the two experiments
agree? Can you use this information to gauge how
long it would take to replicate the entire genome?
- It seems like the second replication progressed slower in the beginning before 3H-thymidine was added. After that, the length of labeled and weakly labeled parts looks comparable to the one of experiment one.
- It seems like the weakly labeled ends are approaching each other in both experiments. This suggests that a full replication would take about half an hour longer in both cases.

5–12 If you compare the frequency of the 16 possible
dinucleotide sequences in the E. coli and human genomes,
there are no striking differences except for one dinucleo-
tide, 5′-CG-3′. The frequency of CG dinucleotides in the
human genome is significantly lower than in E. coli and
significantly lower than expected by chance. Why do you
suppose that CG dinucleotides are underrepresented in
the human genome?
- Maybe because they don't bend as well around histones?
- Maybe because they are harder to pull apart?
- Looked up solution: It's because 5'-CG-3' increases the changes of cytosine methylation, which increases the chances of it deaminating to thymine. → CG turns into TG
- It doesn't happen in E. coli because they methylate different than vertebrates

5–13 With age, somatic cells are thought to accumulate
genomic “scars” as a result of the inaccurate repair of dou-
ble-strand breaks by nonhomologous end joining (NHEJ).
Estimates based on the frequency of breaks in primary
human fibroblasts suggest that by age 70, each human
somatic cell may carry some 2000 NHEJ-induced muta-
tions due to inaccurate repair. If these mutations were
distributed randomly around the genome, how many pro-
tein-coding genes would you expect to be affected? Would
you expect cell function to be compromised? Why or
why not? (Assume that 2% of the genome—1.5% protein-
coding and 0.5% regulatory—is crucial information.)
- 2000 * 0.02 = 40 critical mutations per cell
- 40 * 1.5/2 = 30
- → on average a cell should have 30 mutations in protein-coding cells at the age of 70. This could lead to impairment, but much of it is balanced because cells are diploid, so if one gene on one of the chromosomes is damaged, the other one can still carry out this function

5–14 Draw a schematic diagram of the double Holliday
junction that would result from strand invasion by both
ends of the broken duplex into the intact homologous
duplex shown in Figure Q5–3. Label the left end of each
strand in the Holliday junction 5′ or 3′ so that the rela-
tionship to the parent and recombinant duplexes is clear.
Indicate how DNA synthesis would be used to fill in any
single-strand gaps in your double Holliday junction.

5–15 In addition to correcting DNA mismatches, the
mismatch repair system acts to discourage homologous
recombination between DNA duplexes that are only mod-
erately similar in sequence. Why would recombination
between moderately similar sequences pose a problem for 
human cells?
- It might lead to the loss of an entire gene that is critical fur function and thus be lethal.

5–16 Cre recombinase is a site-specific enzyme that
catalyzes recombination between two LoxP DNA sites. Cre
recombinase pairs two LoxP sites in the same orientation,
breaks both duplexes at the same point in each LoxP site,
and joins the ends with new partners so that each LoxP site
is regenerated, as shown schematically in Figure Q5–4A.
On the basis of this mechanism, predict the arrangement
of sequences that will be generated by Cre-mediated
site-specific recombination for each of the two DNAs
shown in Figure Q5–4B.
- A: "acbd"
- B: "ad" and separate "bc" 

5–17 It is thought that a self-correcting polymerase
cannot start chains de novo because the initial nucleotides
will be weakly paired, thus subject to removal by an effi-
cient proofreading exonuclease. This argument implies
the converse: an enzyme that starts chains anew cannot
be efficient at self-correction. Remarkably, SARS-CoV-2
encodes an RNA replicase complex that is able to do both:
using RNA as a template, it starts strands de novo and
removes errors with an efficient 3′-to-5′ proofreading exo-
nuclease. How might this be possible?
- Maybe it has a part that can produce primers and a part that gets activated later, that can elongate with a primer and uses proofreading
