
##### True/False
Each strand in a β sheet is a helix with two amino
3–1 acids per turn.
→ False!

3–2 Loops of polypeptide that protrude from the surface of a protein often form the binding sites for other molecules.
→ True

3–3 An enzyme reaches a maximum rate at high substrate concentration because it has a fixed number of active sites where substrate binds.
→ Explanation is weird, but conclusion is true - It reaches V_max because the process of catalyzing the enzyme is not perfectly instantaneously and diffusion of enzymes into the active site isn't infinitely fast

3–4 Higher concentrations of enzyme give rise to a higher turnover number for that enzyme.
→ false - turnover number is fixed (max number of reaction/second enzyme could catalyze when always saturated with substrate)

3–5 Enzymes that undergo cooperative allosteric transitions invariably consist of symmetrical assemblies of multiple subunits.
→ Not all of them, but most

3–6 Continual addition and removal of phosphates by protein kinases and protein phosphatases is wasteful of energy—because their combined action consumes ATP— but it is a necessary consequence of effective regulation by phosphorylation.
→ True

##### Discuss the following problems.

3–7 Titin, which has a mass of about 3 × 106 daltons, is the largest polypeptide yet described. Titin molecules extend from muscle thick filaments to the Z disc; they are thought to act as springs to keep the thick filaments centered in the sarcomere. Titin is composed of a large number of repeated immunoglobulin (Ig) sequences of 89 amino acids, each of which is folded into a domainabout 4 nm in length  You suspect that the springlike behavior of titin is caused by the sequential unfolding (and refolding) of individual Ig domains. You test this hypothesis using an atomic force microscope, which allows you to pick up one end of a protein molecule and pull with an accurately measured force. For a fragment of titin containing seven repeats of the Ig domain, this experiment gives the sawtooth force-versus-extension curve shown in Figure Q3–1B. If the experiment is repeated in a solution of 8 M urea (a protein denaturant), the peaks disappear and the measured extension becomes much longer for a given force. If the experiment is repeated after the protein has been cross- linked by treatment with glutaraldehyde, once again the peaks disappear but the extension becomes much smaller for a given force.

A. Are the data consistent with your hypothesis that titin’s springlike behavior is due to the sequential unfolding of individual Ig domains? Explain your reasoning.
- It seems like it's consistent because there are many unfolds at similar force levels which suggests, that there is a gradual unfolding of one after the other

B. Is the extension for each putative domain- unfolding event the magnitude you would expect? (In an extended polypeptide chain, amino acids are spaced at intervals of 0.34 nm.)
- 89AAs * 0,34nm  = 30,26 → Each unfolding event should add about 30nm of extension. the graph looks very consistant with this

C. Why does the force collapse so abruptly after each peak?
- As soon as the force is high enough to "stretch appart" one of the domains, it doesn't pull hard any more because the bonds are broken - the force is just needed to get them apart. Keeping them apart requires way less force

3-8 Consider the following statement. “To produce one molecule of each possible kind of polypeptide chain, 300 amino acids in length, would require more atoms than exist in the universe.” Given the size of the universe, do you suppose this statement could possibly be correct? Because counting atoms is a tricky business, consider the problem from the standpoint of mass. The observable universe is estimated to contain 1080 protons plus neutrons, whichhave a total mass of about 1.7 × 1056 grams. Assuming that

(110 * 20^300 /1,66053906892 * 10^-24) - 1,7 * 10^56 = +∞
→ yes, it's way greater


3–9 The so-called kelch motif consists of a four-stranded β sheet shaped like the blade of a propeller. It is usually found to be repeated four to seven times, forming a β propeller, or kelch repeat domain, in a multidomain protein. One such kelch repeat domain is shown in Figure Q3–2. Would you classify this domain as an in-line or plug-in type domain?
→ in-line because it's a continuous amino acid sequence that makes it up


3–10 In principle, dimers of identical proteins could be arranged either “head-to-tail” (same as “tail-to-head”) or “head-to-head” (equivalent to “tail-to-tail”), as illusMBoC7 mQ3.1/Q3.02 trated schematically in Figure Q3–3. Do you suppose that one type of dimer is significantly more common than the other? Why or why not?
→ probably head to tail is most common because it can form long chains what head to head can't


3-11 An antibody binds to another protein with an equilibrium constant, K, of 5 × 109 M–1. When it binds to a second, related protein, it forms three fewer hydrogen bonds, reducing its binding affinity by 11.9 kJ/mole. What is the K for its binding to the second protein? 
\[Free-energy change is related to the equilibrium constant by the equaMBoC7 nQ3.101/Q3.03 tion ∆G° = –2.3 RT log K, where R is 8.3 × 10–3 kJ/(mole K) and T is 310 K.]
→ K = 5x10^7

3-12 3–12 In bacteria, the protein SmpB binds to a special species of tRNA, tmRNA, to eliminate the incomplete proteins made from truncated mRNAs. If the binding of SmpB to tmRNA is plotted as fraction tmRNA bound versus SmpB concentration, one obtains a symmetrical S-shaped curve as shown in Figure Q3–4. This curve is a visual display of a very useful relationship between Kd and concentration, which has broad applicability. The general expression for fraction of ligand (L) bound to a protein (Pr) is derived from the equation for Kd (Kd = [Pr][L]/[Pr–L]) by substituting ([L]TOT – [L])for [Pr–L] and rearranging. Because the total concentration of ligand ([L]TOT) is equal to the free ligand ([L]) plus bound ligand ([Pr–L]), fraction bound = [Pr–L]/[L]TOT = [Pr]/([Pr] + Kd) For SmpB and tmRNA, the fraction bound = [SmpB–tmRNA]/ [tmRNA]TOT = [SmpB]/([SmpB] + Kd). Using this relationship, calculate the fraction of tmRNA bound for SmpB MBoC7 mQ3.03/Q3.04 concentrations equal to 104 Kd, 103 Kd, 102 Kd, 101 Kd, Kd, 10–1 Kd, 10–2 Kd, 10–3 Kd, and 10–4 Kd.
10^4 Kd → 1 / (1 + 10^4) = 0.00009999
10^3 Kd → 1 / (1 + 10^3) = 0.000999
10^2 Kd → 1 / (1 + 10^2) = 0.00990099
10^1 Kd → 1 / (1 + 10^1) = 0.09090909
Kd      → 1 / (1 + 1)    = 0.5
10^-1 Kd → 1 / (1 + 10^-1) = 0.90909091
10^-2 Kd → 1 / (1 + 10^-2) = 0.99009901
10^-3 Kd → 1 / (1 + 10^-3) = 0.999001
10^-4 Kd → 1 / (1 + 10^-4) = 0.99990001

3–13 The enzyme hexokinase adds a phosphate to d-glucose but ignores its mirror image, l-glucose. Suppose that you were able to synthesize hexokinase entirely from d-amino acids, which are the mirror image of the normal l-amino acids. 
A. Assuming that the “d” enzyme would fold to a stable conformation, what relationship would you expect it to have to the normal “l” enzyme? 
→ It would probably fold in a way that it forms a mirror image of the normal L-enzyme
B. Do you suppose the “d” enzyme would add aphosphate to l-glucose and ignore d-glucose?
→ Because the "d" enzymes active site is mirrored as well, it should now work on L-glucose as it worked before on d-glucose

3–14 Many enzymes obey simple Michaelis–Menten kinetics, which are summarized by the equation rate = Vmax[S]/([S] + Km) where Vmax substrate, and Km = maximum velocity, [S] = concentration of = the Michaelis constant. It is instructive to plug a few values of [S] into the equation to see how rate is affected. What are the rates for [S] equal to zero, equal to Km, and equal to infinite concentration?
→ for =0 the equation goes to infinity
→ for =K_m the equation is V_max/2
→ for =Infinite, the equation would become V_max

3–15 Synthesis of the purine nucleotides AMP and GMP proceeds by a branched pathway starting with ribose 5-phosphate (R5P), as shown schematically in Figure Q3–5. Using the principles of feedback inhibition, propose a regulatory strategy for this pathway that ensures an adequate supply of both AMP and GMP and minimizes the buildup of the intermediates (A–I) when supplies of AMP and GMP are adequate.
→ AMP should inhibit the step E→F
→ GMP should inhibit the step E→H
→ AMP & GMP together should inhibit R5P→A (when they are both present in sufficient quantities)

3–16 How do you suppose that a molecule of hemoglobin is able to bind oxygen efficiently in the lungs and yet MBoC7 mQ3.04/Q3.05 release it efficiently in the tissues?
→ I could imagine that hemoglobin undergoes cooperative allosteric transition when it binds to one oxygen, making it more easy for more oxygen to bind and more stable to keep it bound like this. That way it's "hard" to take up the first oxygen, but as soon as this is in, taking more or keeping all 4 bounded becomes "easy". This happens in the lungs due to high oxygen concentration and is only reversed later in tissue where oxygen concentration is low. It also help with release, because as soon as some oxygen it released, it changes shape so release is easier.

3–17 Rous sarcoma virus (RSV) carries an oncogene called Src, which encodes a continually active protein tyrosine kinase that leads to unchecked cell proliferation. Normally, Src carries an attached fatty acid (myristoylate) group that allows it to bind to the cytoplasmic side of the plasma membrane. A mutant version of Src that does not allow attachment of myristoylate does not bind to the membrane. Infection of cells with RSV encoding either the normal or the mutant form of Src leads to the same high level of protein tyrosine kinase activity, but the mutant Src does not cause cell proliferation.
A. Assuming that the normal Src is all bound to the plasma membrane and that the mutant Src is distributed throughout the cytoplasm, calculate their relative concentrations in the neighborhood of the plasma membrane. For the purposes of this calculation, assume that the cell is a sphere with a radius (r) of 10 μm and that the mutant Src is distributed throughout the cell, whereas the normal Src is confined to a 4-nm-thick layer immediately beneath the membrane. \[For this problem, assume that the membrane has no thickness. The volume of a sphere is (4/3)πr3.]
- Whole cell volume: (4/3) * 3,14159 * 10^3 ~ 4.189
- Plasma membrane area volume: ((4/3) * 3,14159 * 10^3) - ((4/3) * 3,14159 * (10-0,004)^3) ~ 5
	- → relative concentration = (1/5)/(1/4189) = 837,8 → The concentration of Src when bound to membrane is ~ 84x higher that in cell

B. The target (X) for phosphorylation by Src resides in the membrane. Explain why the mutant Src does not cause cell proliferation.
- There is not enough Src close to X if it's not bound to the membrane. It might phosphorylate some from time to time, but will not reach a consistant threshold amount of phosphorylation
